src/interfaces/store/connectionStore.interface.ts

```diff
@@ -1,12 +1,14 @@
 import { Connection } from "@src/types/models";
 
-export interface ConnectionCheckerStore {
+export interface ConnectionStore {
 	retries: number;
+	creatingConnection: boolean;
```

Comment:
Please select another name for it:
1. isProcessingConnection
2. isConnectionOperationActive
3. connectionInProgress